File: iOS/Mobile/CoursesTabBarViewController.m
//
//  CoursesTabBarViewController.m
//  Mobile
//
//  Created by Jason Hocker on 10/3/12.
//  Copyright (c) 2012 Ellucian. All rights reserved.
//

#import "CoursesTabBarViewController.h"

@interface CoursesTabBarViewController ()

@end

@implementation CoursesTabBarViewController

-(void) viewDidLoad
{
    if([self.tabBar respondsToSelector:@selector(setTranslucent:)]) {
        self.tabBar.translucent = NO;
    }
}

@end

File: iOS/Mobile/CoursesTabBarViewController.swift
//
//  CoursesTabBarViewController.swift
//  Mobile
//
//  Created by Jason Hocker on 10/14/15.
//  Copyright © 2015 Ellucian Company L.P. and its affiliates. All rights reserved.
//

import Foundation

class CoursesTabBarViewController : UITabBarController {
    
    var module : Module?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.tabBar.translucent = false
    }
}